diff -rup acestream-launcher-2.1.0.orig/acestream_launcher/config.py acestream-launcher-2.1.0/acestream_launcher/config.py
--- acestream-launcher-2.1.0.orig/acestream_launcher/config.py	2021-04-28 18:20:54.000000000 +0100
+++ acestream-launcher-2.1.0/acestream_launcher/config.py	2021-09-11 15:56:55.005174668 +0100
@@ -16,7 +16,7 @@ class ConfigHandler(object):
       'host':    '127.0.0.1',
       'port':    '6878',
       'player':  'mpv',
-      'timeout': '30',
+      'timeout': '90',
       'hls': 'False',
       'verbose': 'False'
     })
diff -rup acestream-launcher-2.1.0.orig/acestream_launcher/launcher.py acestream-launcher-2.1.0/acestream_launcher/launcher.py
--- acestream-launcher-2.1.0.orig/acestream_launcher/launcher.py	2021-09-11 15:55:27.295577617 +0100
+++ acestream-launcher-2.1.0/acestream_launcher/launcher.py	2021-09-11 15:56:27.089984644 +0100
@@ -58,6 +58,7 @@ class StreamLauncher(object):
     self.host = config.get('host')
     self.port = config.getint('port')
     self.args = parser.parse_args()
+    print ("You may run: acestream-launcher %s" % self.args.url)
     self.noty = NotifyHandler(self.args.player)
 
   @property
diff -rup acestream-launcher-2.1.0.orig/acestream_launcher/player.py acestream-launcher-2.1.0/acestream_launcher/player.py
--- acestream-launcher-2.1.0.orig/acestream_launcher/player.py	2021-09-11 15:55:27.295577617 +0100
+++ acestream-launcher-2.1.0/acestream_launcher/player.py	2021-09-11 15:56:27.089984644 +0100
@@ -17,6 +17,7 @@ class PlayerHandler(Observable):
   def start(self, url, **kwargs):
     cmargs = self.args
     cmargs.append(url)
+    print("\nYou may run: %s" % " ".join(cmargs))
 
     thread = Thread(target=self._start_process, args=(cmargs), kwargs=kwargs)
     thread.setDaemon(True)
diff -rup acestream-launcher-2.1.0.orig/acestream-launcher.desktop acestream-launcher-2.1.0/acestream-launcher.desktop
--- acestream-launcher-2.1.0.orig/acestream-launcher.desktop	2021-09-11 15:55:27.296577624 +0100
+++ acestream-launcher-2.1.0/acestream-launcher.desktop	2021-09-11 15:56:27.090984651 +0100
@@ -6,8 +6,8 @@ Comment=Open AceStream links with any Me
 Exec=acestream-launcher %u
 TryExec=acestream-launcher
 Icon=multimedia-video-player
-Terminal=false
+Terminal=true
 Type=Application
-MimeType=x-scheme-handler/acestream;
+MimeType=x-scheme-handler/acestream;application/x-acelive;
 X-KDE-Protocols=acestream
 NoDisplay=true
