--- ./acestream_launcher/config.py.orig	2021-02-16 20:31:13.073533581 +0000
+++ ./acestream_launcher/config.py	2021-02-16 20:31:32.457666038 +0000
@@ -16,7 +16,7 @@ class ConfigHandler(object):
       'host':    '127.0.0.1',
       'port':    '6878',
       'player':  'mpv',
-      'timeout': '30',
+      'timeout': '90',
       'verbose': 'False'
     })
 
--- ./acestream_launcher/launcher.py.orig	2021-02-16 20:31:13.073533581 +0000
+++ ./acestream_launcher/launcher.py	2021-02-16 20:31:32.458666045 +0000
@@ -52,6 +52,7 @@ class StreamLauncher(object):
     self.host = config.get('host')
     self.port = config.getint('port')
     self.args = parser.parse_args()
+    print ("You may run: acestream-launcher %s" % self.args.url)
     self.noty = NotifyHandler(self.args.player)
 
   @property
--- ./acestream_launcher/player.py.orig	2021-02-16 20:31:13.074533587 +0000
+++ ./acestream_launcher/player.py	2021-02-16 20:31:32.458666045 +0000
@@ -17,6 +17,7 @@ class PlayerHandler(Observable):
   def start(self, url, **kwargs):
     cmargs = self.args
     cmargs.append(url)
+    print("\nYou may run: %s" % " ".join(cmargs))
 
     thread = Thread(target=self._start_process, args=(cmargs), kwargs=kwargs)
     thread.setDaemon(True)
--- ./acestream-launcher.desktop.orig	2021-02-16 20:31:13.074533587 +0000
+++ ./acestream-launcher.desktop	2021-02-16 20:31:32.458666045 +0000
@@ -6,8 +6,8 @@ Comment=Open AceStream links with any Me
 Exec=acestream-launcher %u
 TryExec=acestream-launcher
 Icon=multimedia-video-player
-Terminal=false
+Terminal=true
 Type=Application
-MimeType=x-scheme-handler/acestream;
+MimeType=x-scheme-handler/acestream;application/x-acelive;
 X-KDE-Protocols=acestream
 NoDisplay=true
